jwt/api.py

```diff
@@ -16,7 +16,7 @@
 
 
 class PyJWT(object):
-    def __init__(self, algorithms=None):
+    def __init__(self, algorithms=None, options=None):
```

Comment:
and to reduce redundancy, the verification_options dict could leave out the `verify_` prefix of the keys.